update blog (#321)
cilium/cilium.io: src/posts/2023-07-20-trendyol/index.md

---
date: '2023-07-26T10:12:00.000Z'
title: 'Unleashing the Power of Cilium CNI to Propel Trendyol’s Performance Up to 40%!'
ogImage: ogimage.webp

> ogSummary: 'Explore the reasons behind Trendyol choice of Cilium as well as the value they got.'

Reviewer: ```suggestion
ogSummary: 'Explore the reasons why Trendyol chose Cilium and the value they received from it'
```
